### DISCUSSION TITLE
Designing for Decentralization

### DISCUSSION THREAD

**Discussion by lottabydesign** [BitcoinDesign/Guide#165] [General]
Fundamentally, people fear what they do not understand and hate what they cannot conquer. 

This is why in January 2021, the Central Bank of Nigeria released a circular banning the exchange of cryptocurrency by financial institutions and directed banks to close accounts trading in crypto leading to an uproar. 

The pressing questions now becomes:

1. How might we fully educate Africans who do not understand and Africans who understand crypto so they can use and keep using respectively? (Demystifying crypto)

2. How might we generate interesting use cases that are truly inclusive and are able to make crypto mainstream?

3. How might we organically increase the viral coefficient of crypto users in Africa? (Currently only 1.4 million out of 1.2 billion people use crypto in Africa.)

4. How might we fully give financial power to the average African through the use of decentralized currency?

**Comment by cbrennanpoole** [comment]
RE: Designing for Decentalization

2. How might we generate interesting use cases that are truly inclusive and are able to make crypto mainstream?
    - I like this. 
        1.  First thoughts:
            - to gain trust – in order to help deliver `trustless`.
        2.  Careful with tone; I believe your intentions are pure... However, 
            - `educate Africans`
            - `users in Africa?`
            - `crypto in Africa`.
            - `average African`. 

> Feels Western. As an American, we come off... off...  I have no clue where you're from or your ethnicity. I'd put some crypto on `not Africa`, and would likely lose like I often do trying to trade rather than accumulate, and average-in. That's what it takes, I believe. As trustless is the ingredient needed to hopefully find trust again.  

(I could be altogether off in sharing the feedback, please take no offense; just sharing my initial thoughts after reading).

**Reply by lottabydesign** [reply]
Thanks for your interesting perspective to this discussion!  And no, I am African :)

**Comment by pavlenex** [comment]
Good topic to bring up FYI guys, we're addressing the `Decentralization` as the part of the `Principles` page, so you can chime in there with feedback as well, since it seems tightly related to the topic.  https://github.com/BitcoinDesign/Guide/pull/144

**Reply by lottabydesign** [reply]
Interesting! Will be sure to check in. Thanks :)

**Comment by johnsBeharry** [comment]
Thanks for sharing this - a presentation by David Huseby comes into mind. He shared 9 problems with decentralised systems. Discovery, Introduction, Coherence, Public Services, Trust, Privacy, Coordination, Membership, Persistent State. He claims if a system does not have all of the above properties it is not decentralised. So then, there are incentives for the creation centralised solutions to solve at those gaps. Examples are bountiful - Github, Coinbase, Paxful.

> How might we fully educate Africans who do not understand and Africans who understand crypto so they can use and keep using respectively? (Demystifying crypto)

- [Hello Bitcoin](https://hellobitco.in/) cc @ConorOkus

> How might we generate interesting use cases that are truly inclusive and are able to make crypto mainstream?

There is a time dimension that needs to be considered for this one.
1. Applied to current uses cases, bitcoin can solve immediate problems.
2. Though experimentation bitcoin be used in innovations of new products/systems.

It could take economies and fiat collapsing to drive mainstream adoption, thats quite dark cause people would end up getting hurt even if it may prove out the use case of bitcoin in a very real way. Or on the lighter side it could take some innovation that showcases these properties in a novel way. Which has the most impact for mainstream adoption, not sure how we are to determine this, but both happening in parallel is what we (as in ecosystem) will end up doing.

> How might we organically increase the viral coefficient of crypto users in Africa? (Currently only 1.4 million out of 1.2 billion people use crypto in Africa.)

Seems like its already lifting off - https://bitcoinmagazine.com/articles/nigerians-need-bitcoin-so-where-will-the-restrictions-lead

One of the biggest issues in these regions is still access though, besides these bans, identity verification that's required on centralised exchanges where most people go to acquire bitcoin first. As you stated the government is trying to regulate away bitcoin use but thats why decentralised p2p markets like [Bisq](https://bisq.network/
) are so important. As you stated in the OP the solution to the government's reaction is to go peer-to-peer (> decentralisation) this will only strengthen the resiliency of bitcoin and increase its accessibility.

> How might we fully give financial power to the average African through the use of decentralized currency?

What does it mean to have financial power?

**Reply by lottabydesign** [reply]
Great! Having financial power in this perspective includes ways people can create economic growth for themselves because of their crypto knowledge
